brigade.js

```diff
@@ -66,14 +69,36 @@ function test() {
   return job;
 }
 
+function testViaDocker() {
+  // Create a new job to run Go tests via Docker
+  // to ensure Docker-based builds are functional
+  var job = new Job("tests-via-docker", "docker:stable-dind");
+  // Set privileged true to enable Docker-in-Docker
+  job.privileged = true;
+  // Set a few environment variables.
+  job.env = {
+    "GO111MODULE": "on",
+    "DOCKER_INTERACTIVE": "false",
+  };
+  // Run Go unit tests
+  job.tasks = [
+    "apk add --update --no-cache make git > /dev/null",
+    "dockerd-entrypoint.sh > /dev/null 2>&1 &",
+    "sleep 20",
```

Comment:
What is the magic 20 second sleep waiting for? Can we add a comment for this line please? 😀

Reply:
✨ Comment added